README.md

```diff
@@ -98,7 +98,7 @@ extern crate crossbeam;
 
 ## Compatibility
 
-The minimum supported Rust version is 1.28. Any change to this is considered a breaking change.
+The minimum supported Rust version is 1.36. Any change to this is considered a breaking change.
```

Comment:
> Any change to this is considered a breaking change.

... then it should be accompanied by a semver bump of all affected crates.

Reply:
(or at least update the text if the policy has changed)

Reply:
Yeah, we decided bumping MSRV is a breaking change: https://github.com/crossbeam-rs/crossbeam/commit/d953f49ecc7eb5fe6c271b018a4a753ba435ee58  We need to write a changelog...

Reply:
Done.